player2_skill_factory/player2_factory.py

```
@@ -0,0 +1,313 @@
+        counts = {table: con.execute(f"SELECT COUNT(*) FROM {table}").fetchone()[0] for table in ["repos", "model_lanes", "skills", "tasks", "evidence"]}
+    print(json.dumps({"db": str(DB_PATH), "counts": counts}, indent=2))
+
+
+def main() -> None:
+    parser = argparse.ArgumentParser(description="Player 2 Skill Factory")
+    parser.add_argument("command", choices=["init", "generate-skills", "score", "trading-map", "proof", "report"])
+    parser.add_argument("--intent", default="build next highest-value automation")
+    args = parser.parse_args()
+
+    if args.command == "init":
+        init_db(); ingest_config()
```

Ruff (0.15.20)

[error] Multiple statements on one line (semicolon)

(E702)